shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/rule/builder/DefaultSQLParserRuleConfigurationBuilderTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.parser.rule.builder;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertFalse;
+import static org.hamcrest.Matchers.is;
+
+import org.apache.shardingsphere.parser.config.SQLParserRuleConfiguration;
+import org.apache.shardingsphere.parser.constant.SQLParserOrder;
+import org.junit.Test;
+
+public final class DefaultSQLParserRuleConfigurationBuilderTest {
+    
+    @Test
+    public void assertBuild() {
+        SQLParserRuleConfiguration actualResult = new DefaultSQLParserRuleConfigurationBuilder().build();
```

Comment:
Can you rename actualResult to actual?

Reply:
done, https://github.com/apache/shardingsphere/pull/21291/commits/c32b8827ed1f09965d04a8f07f3003e89c9ddb2d